shepherd/config.py

```diff
@@ -89,7 +89,7 @@ def replace_env_vars(matchobj):
     ruamel.yaml.add_constructor('!env', env_constructor)
 
     # construct config object
-    config_object = ruamel.yaml.load(config_stream)
+    config_object = ruamel.yaml.load(config_stream, Loader=ruamel.yaml.Loader)
```

Comment:
What does this do?

Reply:
fun stuff - ruamel yaml also implemented `yaml 5.0` safeloads... standard `load` results in warning stating that you should use either `safeload` (can't because of envs) or this update.